contentstore/tests.py

```diff
@@ -29,7 +29,45 @@ def setUp(self):
         self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
 
 
-class TestAnalytics(AuthenticatedAPITestCase):
+class TestContentStore(AuthenticatedAPITestCase):
+
+    def make_schedule(self, minute="0", hour="1", day_of_week="*",
+                      day_of_month="*", month_of_year="*"):
+        schedule, created = Schedule.objects.get_or_create(
+            minute=minute, hour=hour, day_of_week=day_of_week,
+            day_of_month=day_of_month, month_of_year=month_of_year)
+        return schedule
+
+    def make_messageset(self, default_schedule, short_name="new set",
+                        next_set=None):
+        message_set, created = MessageSet.objects.get_or_create(
+            short_name=short_name, next_set=next_set,
+            default_schedule=default_schedule)
+        return message_set
+
+    def make_message(self, messageset, sequence_number=1, lang="eng_GB",
+                     text_content="Testing 1 2 3", binary_content=None):
+        message, created = Message.objects.get_or_create(
+            messageset=messageset, sequence_number=sequence_number,
+            lang=lang, text_content=text_content,
+            binary_content=binary_content)
+        return message
+
+    def make_binary_content(self):
+        # models.generate_new_filename = lambda *a: "20151201010101012345.png"
+        simple_png = BytesIO(
+            b'\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00\x00\x01\x00\x00\x00\x01\x08\x06\x00\x00\x00\x1f\x15\xc4\x89\x00\x00\x00\rIDATx\x9cc````\x00\x00\x00\x05\x00\x01\xa5\xf6E@\x00\x00\x00\x00IEND\xaeB`\x82')   # flake8: noqa
+        simple_png.name = 'test.png'
```

Comment:
I'd just read a sample `.png` from disk and use `pkg_resources.resource_stream('pkg_name', 'filename')` to get it from this package's location on disk.